SDPi_Supplement/asciidoc/volume2/gateways/tf2-ch-b-gateway-dec.adoc

```diff
@@ -1,6 +1,1080 @@
+
+* `<Lower Limit>` is set to *pm:LimitAlertConditionDescriptor/pm:MaxLimits/@Lower* and
+* `<Upper Limit>` is set to *pm:LimitAlertConditionDescriptor/pm:MaxLimits/@Upper*
+
+.Notes
+[%collapsible]
+====
+NOTE: Note that the decimal number needs to be formatted according to the HL7 numeric value formatting rules.
+====
+****
+
+
+====== OBX-8 Abnormal Flags
+
+The OBX-8 field is not required to be set if the gateway exports valid and validated metric data only - this is highly recommended.
+
+.R8027
+[sdpi_requirement#r8027,sdpi_req_level=shall]
+****
+If a <<actor_somds_dec_gateway>> is intended to export metrics with a different validity state (*pm:AbstractMetricValue/pm:MetricQuality/@Validity*) than valid or validated, the measurement status shall be set in this field as specified in the <<ihe_pcd_tf_2_2019>>.
+****
+
+====== OBX-11 Observation Result Status
+
+.R8028
+[sdpi_requirement#r8028,sdpi_req_level=shall]
+****
+For a device-related element such as MDS, VMD, or CHANNEL, a <<actor_somds_dec_gateway>> shall set the OBX-11 field to *"X"*.
+****
+
+.R8029
+[sdpi_requirement#r8029,sdpi_req_level=shall]
+****
+For metrics with the *pm:AbstractMetricValue/pm:MetricQuality/@Validity* set to *Vld* (Valid), a <<actor_somds_dec_gateway>> shall set the OBX-11 field to *"R"*.
+****
+
+.R8030
+[sdpi_requirement#r8030,sdpi_req_level=shall]
+****
+For metrics with the *pm:AbstractMetricValue/pm:MetricQuality/@Validity* set to *Vldated* (Validated Data), a <<actor_somds_dec_gateway>> shall set the OBX-11 field to *"F"*.
+****
+
+====== OBX-14 Date/Time of Observation
+
+.R8031
+[sdpi_requirement#r8031,sdpi_req_level=shall]
+****
+A <<actor_somds_dec_gateway>> shall set the OBX-14 field to the date and time of the intermittently measured metric value.
+
+.Notes
+[%collapsible]
+====
+NOTE: Intermittently measured metrics have the *pm:AbstractMetricDescriptor/@MetricAvailability* set to "*Intr*".
+
+NOTE: <<ref_tbl_dec_obx14_mapping>> defines the mapping of the SDC metric measurement timestamp to the data fields of the HL7 data type *DTM* used in the OBX-14 field.
+====
+****
+
+[#ref_tbl_dec_obx14_mapping]
+.OBX-14 Metric Measurement Timestamp Mapping
+|===
+|HL7 Field |HL7 Component Name |SDC Attribute/Element |Comments
+
+|OBR-14/DTM-1
+|Date/Time
+|pm:EnumStringMetricState
+/pm:MetricValue
+/@DeterminationTime
+
+pm:NumericMetricState
+/pm:MetricValue
+/@DeterminationTime
+
+pm:StringMetricState
+/pm:MetricValue
+/@DeterminationTime
+|Note that the HL7 date & time format differs from the xsd date/time formats and requires a mapping accordingly (see also <<ref_expl_dt_mapping>>).
+
+|===
+
+====== OBX-16 Responsible Observer
+
+.R8032
+[sdpi_requirement#r8032,sdpi_req_level=shall]
+****
+A <<actor_somds_dec_gateway>> shall set the OBR-10 field to the operator.
```

Comment:
Shouldn't this be "... OBX-16 field.."?
Should the same info be entered here as in OBR-10?

Reply:
Yes, it is the same information but OBX-16 is limited to the ID number, and optionally the Family Name and Given Name, OBR-10 may contain more information. I added this in the note of the requirement to make this clear.

Reply:
Then OBR-10 should be replaced by OBX-16: A <<actor_somds_dec_gateway>> shall set the OBX-16 field to the operator.

Reply:
Good point! Fixed!